### ISSUE TITLE
Improve errors when passing unknown local variable

### ISSUE THREAD

**Issue by patryk4815** [pwndbg/pwndbg#2513]
lldb:
<img width="942" alt="image" src="https://github.com/user-attachments/assets/f62cf440-0f50-4016-ba81-e368f4cd2976">

gdb:
<img width="556" alt="image" src="https://github.com/user-attachments/assets/00170c3f-aabc-4922-be1f-732330c4af95">

**Comment by disconnect3d** [comment]
Just to be sure, this is with `set exception-verbose on` right? I believe normally/by default the error is more nice (at least on gdb)?

**Comment by patryk4815** [comment]
yes `exception-verbose on`

**Comment by IamAbhinav03** [comment]
Hi, I am new to open source, I would like to take up this issue if nobody is working on it. Can anyone guide me a little bit on how to start contributing to this repo?

**Comment by patryk4815** [comment]
fixed in https://github.com/pwndbg/pwndbg/pull/2711 by @k4lizen 

```
pwndbg-lldb> jemalloc_find_extent ptr
usage: jemalloc_find_extent [-h] addr
jemalloc_find_extent: error: argument addr: debugger couldn't resolve argument 'ptr': error: <user expression 1>:1:1: use of undeclared identifier 'ptr'
    1 | ptr
      | ^
```

```
pwndbg> jemalloc_find_extent ptr
usage: jemalloc_find_extent [-h] addr
jemalloc_find_extent: error: argument addr: debugger couldn't resolve argument 'ptr': No symbol "ptr" in current context.
```

**Comment by Srishti-Nagpure** [comment]
@patryk4815 I can solve your code can you assign me..

**Comment by disconnect3d** [comment]
No. Please stop spamming. It is open source, you can just send a PR with a fix if something doesn't work. This issue is closed since it was fixed
